pynoc/cisco.py

```diff
@@ -1,10 +1,32 @@
 """Cisco Switch control object."""
 
 import logging
+import warnings
 
-from six import PY3
 from netaddr import EUI, mac_unix_expanded
-from paramiko import SSHClient, AutoAddPolicy
+from netmiko import ConnectHandler
+
+
+def deprecated(func):
+    """Deprecated decorator.
```

Comment:
Phrasing of this docstring is failing the build at FLAKE8: `pynoc/cisco.py:10:1: D401 First line should be in imperative mood; try rephrasing`. Perhaps rephrase to "Decorates a deprecated function to warn when it is called."